src/components/app.jsx

```diff
@@ -26,4 +27,13 @@ function App() {
   );
 }
 
+/**
+ * Checks if user is logged in
+ 
```

Comment:
Add param and return value (if there is a return) for the function to the docstring here.